allauth/socialaccount/providers/douban/provider.py

```diff
@@ -12,7 +12,7 @@ def get_avatar_url(self):
 
     def to_str(self):
         dflt = super(DoubanAccount, self).to_str()
-        return self.account.extra_data.get("name", dflt)
+        return self.account.extra_data.get("uid", dflt)
```

Comment:
As above, why not keep `name` ?

Reply:
It looked like `name` might not be unique, but the alphanumeric `uid` might be unique and recognisable as a username. I haven't tested this with a real Douban account.